deepfence_worker/cronjobs/neo4j.go

```diff
@@ -197,11 +197,13 @@ func CleanUpDB(ctx context.Context, task *asynq.Task) error {
 
 	// Delete old with no data
 	if _, err = session.Run(`
-		MATCH (n:Node)
+		MATCH (n:Node) <-[:HOSTS]- (cr:CloudRegion)
 		WHERE n.active = false
 		AND NOT exists((n) <-[:SCANNED]-())
 		OR n.updated_at < TIMESTAMP()-$old_time_ms
-		WITH n LIMIT 10000
+		WITH n, cr LIMIT 10000
+		SET cr.cr_shown = CASE WHEN n.kubernetes_cluster_id= "" or n.kubernetes_cluster_id is null THEN cr.cr_shown - 1 ELSE cr.cr_shown END,
```

Comment:
This one is tricky - the `cr_shown` refers to `Cloud Resource` being shown.
But now we are also adding `Node` to the mix.
Let's change `cr_shown` to `res_shown`